library/core/class.oauth2.php

```diff
@@ -373,9 +377,9 @@ public function settingsEndpoint($sender, $args) {
         $formFields = [
             'AssociationKey' =>  ['LabelCode' => 'Client ID', 'Description' => 'Enter the unique ID of the authentication application.'],
```

Comment:
Since the description describe the field we can probably remove all the "Enter the" in there.